### PR TITLE
CSP - Added new function to remove duplicates from list

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/47039)

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -1788,6 +1788,22 @@ def argToList(arg, separator=','):
     return [arg]
 
 
+def remove_duplicates_from_list_arg(args, field):
+    """
+        Removes duplicates from a dict after calling argToList.
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1793]:
```suggestion
        Removes duplicates from a list after calling argToList.
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -1788,6 +1788,22 @@ def argToList(arg, separator=','):
...
+        For example: args: {'ids': "1,2,1"} ,  field='ids'
+        The return output will be ["1","2"]
+        :type args: dict
+        :param args: Args to be converted (required)
+        :type field: str
+        :param field: Field in args to be converted into list without duplicates (required)
+       :return: A python list of args without duplicates
+       :rtype: ``list``
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1801]:
```suggestion
        :return: A python list of args without duplicates
        :rtype: ``list``
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -1788,6 +1788,22 @@ def argToList(arg, separator=','):
     return [arg]
 
 
+def remove_duplicates_from_list_arg(args, field):
+    """
+        Removes duplicates from a dict after calling argToList.
+        For example: args: {'ids': "1,2,1"} ,  field='ids'
+        The return output will be ["1","2"]
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1795]:
```suggestion
        For example: args: {'ids': "1,2,1"}, field='ids'
        The return output will be ["1", "2"]

```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 4]
```diff
@@ -1788,6 +1788,22 @@ def argToList(arg, separator=','):
...
+        :type args: dict
+        :param args: Args to be converted (required)
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1797]:
```suggestion
        :param args: Args to be converted (required)

```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 5]
```diff
@@ -1788,6 +1788,22 @@ def argToList(arg, separator=','):
...
+        :type field: str
+        :param field: Field in args to be converted into list without duplicates (required)
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1799]:
```suggestion
        :param field: Field in args to be converted into list without duplicates (required)

```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 6]
```diff
@@ -1788,6 +1788,22 @@ def argToList(arg, separator=','):
     return [arg]
 
 
+def remove_duplicates_from_list_arg(args, field):
+    """
+        Removes duplicates from a dict after calling argToList.
+        For example: args: {'ids': "1,2,1"} ,  field='ids'
+        The return output will be ["1","2"]
+        :type args: dict
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1796]:
```suggestion
        :type args: ``dict``
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 7]
```diff
@@ -1788,6 +1788,22 @@ def argToList(arg, separator=','):
...
+        :param args: Args to be converted (required)
+        :type field: str
```
Comment (yaakovi) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1798]:
```suggestion
        :type field: ``str``
```

File: Packs/Base/ReleaseNotes/1_18_13.md [hunk 8]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- Add new function for removing duplicates after using **argToList**.
```
Comment (yaakovi) [inline Packs/Base/ReleaseNotes/1_18_13.md:4]:
```suggestion
- Added the **remove_duplicates_from_list_arg** function for removing duplicated items in list arguments.
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 9]
```diff
@@ -1788,6 +1788,25 @@ def argToList(arg, separator=','):
...
+        :param args: Args to be converted (required)
+
+        :type field: ``str``
+        :param field: Field in args to be converted into list without duplicates (required)
+
+        :return: A python list of args without duplicates
+        :rtype: ``list``
+    """
+    convert_to_list = argToList(args.get(field))
+    return list(set(convert_to_list))
```
[the remark this answers is not in the document]
Comment (abaumgarten) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1807]: Actually, I don't see the value in adding a dummy UT to test python functionally...

**Review by yaakovi — APPROVED** [review]

**Pull request merged**